src/bosh-director/spec/unit/jobs/update_stemcell_spec.rb

```diff
@@ -324,6 +326,77 @@
       end
     end
 
+    context 'when information about stemcell formats is not enough' do
```

Comment:
The descriptions of the`it` blocks in this `context` should be updated to convey the assertion in the test. Currently they read like nested `context`, and give no clear indication of what the test is trying to prove.